lib/utils/format-search-stream.js

```diff
@@ -16,7 +15,10 @@ const ansiTrim = require('strip-ansi')
 // The returned stream will format this package data
 // into a byte stream of formatted, displayable output.
 
-module.exports = (opts) => {
+let stripAnsi
+module.exports = async (opts) => {
+  stripAnsi = await import('strip-ansi')
+  stripAnsi = stripAnsi.default
```

Comment:
To avoid needing to hoist `strip-ansi` what if it was passed along with the opts:

```diff
diff --git a/lib/utils/format-search-stream.js b/lib/utils/format-search-stream.js
index 766028bdd..444c86df8 100644
--- a/lib/utils/format-search-stream.js
+++ b/lib/utils/format-search-stream.js
@@ -15,11 +15,9 @@ const columnify = require('columnify')
 // The returned stream will format this package data
 // into a byte stream of formatted, displayable output.
 
-let stripAnsi
 module.exports = async (opts) => {
-  stripAnsi = await import('strip-ansi')
-  stripAnsi = stripAnsi.default
-  return opts.json ? new JSONOutputStream() : new TextOutputStream(opts)
+  const { default: stripAnsi } = await import('strip-ansi')
+  return opts.json ? new JSONOutputStream() : new TextOutputStream({ ...opts, stripAnsi })
 }
 
 class JSONOutputStream extends Minipass {
@@ -142,7 +140,7 @@ function highlightSearchTerms (str, terms) {
   return colorize(str).trim()
 }
 
-function normalizePackage (data, opts) {
+function normalizePackage (data, { stripAnsi }) {
   return {
     name: stripAnsi(data.name),
     description: stripAnsi(data.description ?? ''),
```